devops/docker/docker-compose.client.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.0"
+services:
+  client:
+    image: kitware/esimmon-client
+    depends_on:
+      - girder
+    build:
+      context: ../../
+      dockerfile: devops/docker/nginx/Dockerfile
+      args:
+        VUE_APP_FOLDER_ID: $VUE_APP_FOLDER_ID
+    ports:
+      - 80:80
```

Comment:
We may want to use say 9090 on the host machine so ```9090:80```

Reply:
Fixed